ArchUnitNETTests/Fluent/PlantUml/_snapshots/PlantUmlFluentComponentDiagramTests_ComponentDiagramFromCustomDependenciesTest.json

```diff
@@ -0,0 +1,3 @@
+{
+  "AutoGenerated": "@startuml\r\nclass \"d\" {\r\n}\r\n[a] --|> [b]\r\n[b] --|> [c]\r\n[c] --|> [a]\r\n@enduml\r\n"
```

Comment:
I'm wondering if this snapshot test will fail on a platform with different line endings 🤔

Reply:
Hmm, unless the way the plantuml file is built changes - only in that case the test will fail

Reply:
> I'm wondering if this snapshot test will fail on a platform with different line endings 🤔

Hmm, unless the way the plantuml file is built changes - only in that case the test will fail

Reply:
Windows uses `\r\n` as newline character sequence, Linux typically just `\n`. I would expect that if you generate the plantuml file in Linux, it would give a different snapshot.